.cookiecutter/includes/requirements/tests.in

```diff
@@ -0,0 +1 @@
+pytest-mock
```

Comment:
`test-pyramid-app` doesn't have our `patch()` fixture. Nowadays we should be using [pytest-mock](https://pytest-mock.readthedocs.io/en/latest/) anyway, there's no point in continuing to maintain our own alternative solution to integrating `mock` and `pytest`.

Again, the cookiecutter should be adding this dependency. It does already add it for packages but apparently not for apps.